src/transformers/models/video_llava/processing_video_llava.py

```diff
@@ -170,15 +170,15 @@ def __call__(
         elif not isinstance(text, list) and not isinstance(text[0], str):
             raise ValueError("Invalid input text. Please provide a string, or a list of strings")
 
-        if encoded_images is not None:
+        if images is not None:
```

Comment:
nit:

I personally would be in favor to define `encoded_images` and `encoded_videos` as `None` at the correct scope. Then here check

> if encoded_images is not None:

This is because within this `if`, it is `encoded_images` being used. Having `if images is not None:` as condition introduces a bit of mental burden when reasoning.

Otherwise, if the original images/videos arguments are no longer needed after `encoded_images` / `encoded_videos`, just name them as `images` / `videos` also works for me.